reactDashboard/src/components/SingleValue/SingleValueBar.css

```diff
@@ -1,5 +1,27 @@
+.region-single-value-bar,
```

Comment:
Maybe add a comment here 'Width independent css attributes' to prevent someone thinking that merging the css blocks would be a good idea

Reply:
comment added